buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -0,0 +1,54 @@
+plugins {
+    `java-library`
+
+    id("otel.spotless-conventions")
+}
+
+group = "io.opentelemetry.contrib"
+version = "1.0.0-alpha"
+
+base.archivesBaseName = "${rootProject.name}-${project.name}"
```

Comment:
Prepending with rootProject looks a bit weird to me. Will check this in a followup

Reply:
Since this was designed for the jmx metric gatherer that runs as an uber jar having the name `opentelemetry-java-contrib-jmx-metrics.jar` seemed more informative and clear to me than just `jmx-metrics`.  This doesn't have to translate to all contributed projects so I'm open to migrating it there.

Reply:
Yeah thinking of making this `opentelemetry-jmx-metrics`